src/main/java/com/netflix/imflibrary/st2067_2/IMFIABConstraintsChecker.java

```diff
@@ -0,0 +1,146 @@
+package com.netflix.imflibrary.st2067_2;
+
+import com.netflix.imflibrary.IMFErrorLogger;
+import com.netflix.imflibrary.IMFErrorLoggerImpl;
+import com.netflix.imflibrary.st0377.header.UL;
+import com.netflix.imflibrary.st2067_201.IABSoundfieldLabelSubDescriptor;
+import com.netflix.imflibrary.utils.DOMNodeObjectModel;
+import com.netflix.imflibrary.utils.ErrorLogger;
+import com.netflix.imflibrary.utils.RegXMLLibDictionary;
+import com.netflix.imflibrary.utils.UUIDHelper;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.netflix.imflibrary.st0377.header.GenericSoundEssenceDescriptor.ElectroSpatialFormulation.MULTI_CHANNEL_MODE;
+import static com.netflix.imflibrary.st2067_201.IABEssenceDescriptor.IMF_IAB_ESSENCE_CLIP_WRAPPED_CONTAINER_UL;
+import static com.netflix.imflibrary.st2067_201.IABEssenceDescriptor.IMMERSIVE_AUDIO_CODING_LABEL;
+
+public class IMFIABConstraintsChecker {
+    private static final Integer IAB_BIT_DEPTH = 24;
+    private static final short IAB_CHANNEL_COUNT = 0;
+
+    public static List<ErrorLogger.ErrorObject> checkIABVirtualTrack(IMFCompositionPlaylistType compositionPlaylistType,
```

Comment:
I believe that this can be moved to the IAB package (_st2067_201_) by just removing the package-private `IMFCompositionPlaylistType` parameter. It is only used to get the EditRate, so it could be replaced with a `Composition.EditRate` parameter instead.
That would keep the IAB code more neatly organized within the same package.

Reply:
Done.